<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>361</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>490</y>
      <width>501</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ON">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 0);</string>
    </property>
    <property name="text">
     <string>INICIAR</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="op1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NRZ-L</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="op1_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NRZ-M</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="op1_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NRZ-S</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="op1_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BIPOLAR RZ</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="op1_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RZ-AMI</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="op1_6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>UNIPOLAR RZ</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="op1_7">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>170</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BI-O-M</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="op1_8">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>220</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BI-O-S</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="op1_9">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BI-O-L</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="op1_10">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>170</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DICODE NRZ</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="op1_11">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>220</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DICODE RZ</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="op1_12">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>120</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DELAY MOD</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="texto">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Myanmar Text</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>674</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="imjj.qrc"/>
  <include location="logo4.qrc"/>
  <include location="Logo1.qrc"/>
  <include location="logo2.qrc"/>
  <include location="logo3.qrc"/>
 </resources>
 <connections/>
</ui>
